src/components/tables/AccessGrants/AccessLinks/Dialog/GenerateInvitation.tsx

```diff
@@ -42,16 +45,42 @@ function GenerateInvitation({
     );
 
     const [prefix, setPrefix] = useState<string>(objectRoles[0]);
+    const [prefixMissing, setPrefixMissing] = useState(false);
+
+    const [suffix, setSuffix] = useState<string>('');
+    const [suffixInvalid, setSuffixInvalid] = useState(false);
+
     const [capability, setCapability] = useState<string>(capabilityOptions[0]);
     const [reusability, setReusability] = useState<string>(typeOptions[0]);
 
     const handlers = {
-        setGrantPrefix: (_event: React.SyntheticEvent, value: string) => {
+        setGrantPrefix: (event: SelectChangeEvent<string>) => {
             if (serverError) {
                 setServerError(null);
             }
 
-            setPrefix(value);
+            const value = event.target.value;
+
+            setPrefixMissing(!hasLength(value));
+
+            const processedValue = value.endsWith('/') ? value : `${value}/`;
```

Comment:
This is identical and needs to be shared. Either in this functions of toss it into a utils.